ff --git a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi.orig b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
index 30f197f..7359aea 100644
--- a/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-wandboard.dtsi
@@ -228,10 +228,16 @@
        fsl,phy_reg_vlev = <0x0294>;
        fsl,phy_reg_cksymtx = <0x800d>;
        status = "okay";
 };
 
+&hdmi_cec {
+        pinctrl-names = "default";
+        pinctrl-0 = <&pinctrl_hdmi_cec_2>;
+        status = "okay";
+};
+
 &i2c1 {
        clock-frequency = <100000>;
        pinctrl-names = "default";
        pinctrl-0 = <&pinctrl_i2c1_1>;
        status = "okay";
